crates/rspack_binding_values/src/dependency.rs

```diff
@@ -178,3 +178,20 @@ impl ToNapiValue for JsDependencyWrapper {
     })
   }
 }
+
+impl FromNapiValue for JsDependencyWrapper {
```

Comment:
JsDependencyWrapper need not impl FromNapiValue

Reply:
Done~